tidb-cloud/serverless-driver-kysely-example.md

```diff
@@ -93,7 +93,7 @@ To complete this tutorial, you need the following:
 
    ```sql
    CREATE TABLE `test`.`person`  (
-     `id` int(11) NOT NULL AUTO_INCREMENT,
+     `id` int NOT NULL AUTO_INCREMENT,
```

Comment:
For create this is OK, but I assume for `SHOW CREATE TABLE` etc. it should be reflected in the correct TiDB Serverless version documentation?